app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,19 @@
+class LikesController < ApplicationController
+  def new
+    @like = Like.new
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+  end
```

Comment:
- [ ] [OPTIONAL] You have done great with the Comments, Posts, and Likes. Everything works fine now but consider removing the 'new' action unless it's needed for rendering a form. You do not need the action to add Likes to a post.